register.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Credard - Register</title>
+  <link rel="stylesheet" href="auth.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+  <nav>
+    <ul>
+      <li><img src="assets/images/logo.png"></li>
+      <li style="float: right;">
+        <div style="display: inline-flex;">
+            <p style="padding: 0px 10px">Already have an account? </p>
+            <a href="/register" class="register_button" style="max-width: 20px !important;">Login</a>
```

Comment:
This is not what we have in design please make corrections
![image](https://user-images.githubusercontent.com/60205816/182849393-ead532e7-af6c-44ef-8636-04f82cd9b127.png)